elyra/__init__.py

```diff
@@ -53,6 +54,7 @@ def _load_jupyter_server_extension(nb_server_app):
          PipelineConfigHandler),
         (url_path_join(web_app.settings['base_url'], r'/elyra/contents/properties/%s' % (path_regex)),
          FileParserHandler),
+        (url_path_join(web_app.settings['base_url'], r'/elyra/validation'), PipelineValidationHandler),
```

Comment:
Seems like this endpoint should be `/elyra/pipeline/validation`.

Reply:
updated